packages/builder-tools/connect-utils/vite-config.ts

```diff
@@ -93,6 +93,8 @@ export function getConnectBaseViteConfig(options: IConnectOptions) {
   const fileEnv = loadEnv(mode, envDir, "PH_");
   const env = { ...fileEnv, ...process.env } as Partial<ConnectEnv>;
 
+  const disableLocalPackages = process.env.DISABLE_LOCAL_PACKAGES === "true";
```

Comment:
Here we should check variables using `env.PH_...` as it includes both the `.env` file and `process.env`, from line 94.